spec/GraphQLOverHTTP.md

```diff
@@ -745,3 +745,37 @@ payload is `application/json` then the client MUST NOT rely on the body to be a
 well-formed _GraphQL response_ since the source of the response may not be the
 server but instead some intermediary such as API gateways, proxies, firewalls,
 etc.
+
+# Additional Notes
+
+## Security
+
+In this specification, GET requests are not supported for mutations due to
+security concerns. GET requests expose variables to logging mechanisms and
+intermediaries due to the URL encoding of parameters, which can lead to
+sensitive data being inadvertently logged. Furthermore, GET requests are
+considered "basic requests" under CORS (Cross-Origin Resource Sharing), meaning
```

Comment:
```suggestion
considered "simple requests" under CORS (Cross-Origin Resource Sharing), meaning
```

It's referred to as "simple", not "basic":

https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#simple_requests